{"name": "test_customer_signIn[edge]", "status": "passed", "attachments": [{"name": " Test Customer sign in is failed ", "source": "bef3338b-080b-428e-a8d0-9c79d7942333-attachment.png", "type": "image/png"}, {"name": "log", "source": "b97c5ba8-4a2e-4139-bb48-2c07fbf2f406-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "init_driver", "value": "'edge'"}], "start": 1647260370405, "stop": 1647260387306, "uuid": "ebffa528-b6ab-4d56-a582-1ff7518115b7", "historyId": "c585e072c0210645023e2a96c76ec875", "testCaseId": "61795a803f78060dfbefbeae26fea2b8", "fullName": "test_customer_signIn.Test_customer_signIn#test_customer_signIn", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "regression"}, {"name": "suite", "value": "test_customer_signIn"}, {"name": "subSuite", "value": "Test_customer_signIn"}, {"name": "host", "value": "LAPTOP-ESSL3DDB"}, {"name": "thread", "value": "2884-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_customer_signIn"}]}